compiler/rustc_hir/src/hir.rs

```diff
@@ -686,7 +685,7 @@ pub struct Crate<'hir> {
     pub attrs: BTreeMap<HirId, &'hir [Attribute]>,
 }
 
-impl Crate<'hir> {
+impl<'hir> Crate<'hir> {
```

Comment:
mirroring the other impls in this crate, we can keep using implicit lifetime declaration

```suggestion
impl Crate<'hir> {
```